client/packages/purchase_orders/src/api/hooks/usePurchaseOrder.ts

```diff
@@ -69,19 +55,33 @@ export const usePurchaseOrder = (id?: string) => {
   const { sortedAndFilteredLines, itemFilter, setItemFilter } =
     useFilteredAndSortedLines(data);
 
+  // DRAFT STATE
+  const [draft, setDraft] = useState<PurchaseOrderFragment | undefined>();
+
+  useEffect(() => {
+    if (data) setDraft(data);
+  }, [data]);
+
+  const handleDraftChange = (input: Partial<PurchaseOrderFragment>) => {
+    if (!draft) return;
+    setDraft({ ...draft, ...input });
+  };
```

Comment:
I've introduced a draft state here to avoid creating separate states for each input that may need to be updated. The Details tab roughly contains around 10 fields, each of which would typically require its own state. This approach simplifies the code